pkg/ebpf/c/common/ksymbols.h

```diff
@@ -1,10 +1,11 @@
-#ifndef __TRACEE_KSYMBOLS_H__
-#define __TRACEE_KSYMBOLS_H__
+#ifndef __COMMON_KSYMBOLS_H__
+#define __COMMON_KSYMBOLS_H__
 
-#include "common/consts.h"
-#include "common/common.h"
-#include <bpf/bpf_helpers.h>
-#include "maps.h"
+#include <vmlinux.h>
+#include <vmlinux_flavors.h>
```

Comment:
I don't see a reason to include this header here

Reply:
ditto (tried to create a pattern, can remove it).